docs/source/examples/petstore/petstore.all.test.rego

```diff
@@ -31,6 +47,43 @@ test_regexp_negative {
 	not deny with input as in
 }
 
+# deny subject group everyone to buy petstore.pet where ctx.age <= 2;
+test_ctx_usage_multiply {
+	in := {
+		"type": "petstore.pet",
+		"verb": "buy",
+		"jwt": sealtest_jwt_encode_sign({
+			"iss": "not_petstore.swagger.io",
+			"sub": "wiley-e-coyote@acme.com",
+			"groups": ["everyone"],
+		}),
+		"ctx": [
+			{"age": 5},
```

Comment:
i don't see this as  valid example - a pet would not have an age of 5 and age of 1 at the same time

Reply:
but it might be an attempt to buy two pets, right? Or it should be two different requests?